web/cypress/e2e/group_1_onboarding/accessibility.spec.ts

```diff
@@ -0,0 +1,86 @@
+import "cypress-axe";
+import { setMobileViewport } from "../../support/helpers/helpers";
+import { onOnboardingPageStartingBusiness } from "../../support/page_objects/onboardingPageNew";
+
+describe("Automated Accessibilty Testing [feature] [all] [group1]", () => {
+  describe("Onboarding Desktop Accessibility Test", () => {
+    beforeEach(() => {
+      cy.loginByCognitoApi();
+      cy.injectAxe();
+    });
+
+    it("Accessibility for All Other Businesses", () => {
+      cy.url().should("include", "onboarding?page=1");
+      onOnboardingPageStartingBusiness.selectBusinessPersonaRadio("STARTING");
+      cy.checkA11y(undefined, {
+        runOnly: {
+          type: "tag",
+          values: ["wcag22aa"],
+        },
+      });
+
+      onOnboardingPageStartingBusiness.getBusinessPersonaRadio("STARTING").should("be.checked");
+      onOnboardingPageStartingBusiness.clickNext();
+
+      cy.url().should("include", "onboarding?page=2");
+      onOnboardingPageStartingBusiness.selectIndustryDropdown("generic");
+      cy.checkA11y(undefined, {
+        runOnly: {
+          type: "tag",
+          values: ["wcag22aa"],
+        },
+      });
+      onOnboardingPageStartingBusiness.clickNext();
+
+      cy.url().should("include", "dashboard");
+      cy.get('[data-testid="header-link-to-profile"]');
+      cy.checkA11y(undefined, {
+        runOnly: {
+          type: "tag",
+          values: ["wcag22aa"],
+        },
+      });
+    });
+  });
+
+  describe("Onboarding Mobile Accessiblility Test", () => {
```

Comment:
Same comment as above
```suggestion
  describe("Onboarding - Mobile", () => {
```